tensorbay/client/segment.py

```diff
@@ -629,6 +645,19 @@ def _generate_frames(self, offset: int = 0, limit: int = 128) -> Generator[Frame
 
         return response["totalCount"]  # type: ignore[no-any-return]
 
+    def _upload_or_import_data(
```

Comment:
Where is import data?

Reply:
Importing auth data in fusion dataset has not been supported yet.